mycore-base/src/main/java/org/mycore/common/xml/MCRURIResolver.java

```diff
@@ -1593,4 +1595,35 @@ public Source resolve(String href, String base) throws TransformerException {
         }
 
     }
+    
+    private static class MCRI18NResolver implements URIResolver {
+        /**
+         * Resolves the I18N String value for the given property.<br><br>
+         * 
+         * Syntax: <code>i18n:{i18n-property}</code>
+         * 
+         * @param href
+         *            URI in the syntax above
+         * @param base
+         *            not used
+         * 
+         * @return the root element of the XML document
+         * @see javax.xml.transform.URIResolver
+         */
+        @Override
+        public Source resolve(String href, String base) throws TransformerException {
+            LOGGER.debug("start resolving " + href);
+            StringTokenizer pst = new StringTokenizer(href, ":", true);
```

Comment:
from Javadocs:
StringTokenizer is a legacy class that is retained for compatibility reasons although its use is discouraged in new code. It is recommended that anyone seeking this functionality use the split method of String or the java.util.regex package instead.